pkgs/tools/misc/mlxbf-bootctl/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mlxbf-bootctl";
+  version = "1.1-6";
+
+  src = fetchFromGitHub {
+    owner = "Mellanox";
+    repo = pname;
+    rev = "${pname}-${version}";
+    hash = "sha256-F49ZZtty+NARXA/doAFLhsQn4XkPW6GWLXGy4waIaM0=";
+  };
+
+  installPhase = ''
+    mkdir -p "$out"/bin
```

Comment:
quotes on `$out` are useless, we know this is a nix store path, it is always proper

Reply:
It doesn't hurt but don't let people believe we need to do it

Reply:
Shellcheck would hate this rhethoric